src/modules/blog/entities/blog-comments.entity.ts

```diff
@@ -0,0 +1,17 @@
+import { Entity, PrimaryGeneratedColumn, Column, ManyToOne } from 'typeorm';
+import { BlogPost } from './blog.entity';
+import { AbstractBaseEntity } from '../../../entities/base.entity';
+import { User } from '../../user/entities/user.entity';
+
+@Entity()
+export class BlogPostComment extends AbstractBaseEntity {
+  @PrimaryGeneratedColumn()
```

Comment:
Remove this, since you're extending the base class and it's not the primary key column

Reply:
fix